src/Security/SslContext.cs

```diff
@@ -523,7 +523,7 @@ public SslStatus SetEncryptionCertificate (SecIdentity identify, IEnumerable<Sec
 			get {
 				IntPtr value;
 				result = SSLCopyPeerTrust (Handle, out value);
-				return (value == IntPtr.Zero) ? null : new SecTrust (value);
+				return (value == IntPtr.Zero) ? null : new SecTrust (value, true);
```

Comment:
@OneMHz it's the **Create** (Copy) rule [1] that applies - that's what the reference to `SSLCopyPeerTrust` meant.  The instance we get is already retained but it's our job to release it. 

That differs from the **Get** rule where we are responsible to retain the instance we get (and also to release it). In both cases we need to release the instance, it's only how we initialize that differs.

[1] https://developer.apple.com/library/content/documentation/CoreFoundation/Conceptual/CFMemoryMgmt/Concepts/Ownership.html#//apple_ref/doc/uid/20001148-103029